workers/unity/Packages/com.improbable.gdk.buildsystem/WorkerBuilder.cs

```diff
@@ -259,10 +259,11 @@ private static void Zip(string zipAbsolutePath, string basePath, bool useCompres
         {
             using (new ShowProgressBarScope($"Package {basePath}"))
             {
-                RedirectedProcess.Run(Common.SpatialBinary, "file", "zip",
-                    $"--output=\"{Path.GetFullPath(zipAbsolutePath)}\"",
-                    $"--basePath=\"{Path.GetFullPath(basePath)}\"", "\"**\"",
-                    $"--compression={useCompression}");
+                RedirectedProcess.Command(Common.SpatialBinary)
```

Comment:
just realized that that used to be part of another PR and was merged in here. Could we please not do this until absolutely necessary? I know you are used to stacks diffs (they are nice with the right infrastructure :) ), but here it just gets more difficult to review a PR when the scope of it suddenly changes

in this case the refactor of the redirectedprocess part could have been merged into the master /develop branch just fine without this part :)